src/services/notification/notification.model.ts

```diff
@@ -0,0 +1,18 @@
+import { Table, Column, Model, DataType } from 'sequelize-typescript'
+import { CommsPayloads } from '../../definitions'
+import { ArrayStringType } from '../../sequelize'
+
+@Table({
+  freezeTableName: true,
+  tableName: 'notifications'
+})
+export default class NotificationModel extends Model {
+  @Column({ allowNull: false })
+  type!: string
+
+  @Column({ ...ArrayStringType('accounts'), allowNull: false })
+  accounts!: string[]
+
+  @Column({ type: DataType.JSON, allowNull: false })
+  payload!: CommsPayloads & { code: string }
```

Comment:
maybe extract type in the definitions just to make it more obvious.

Reply:
Good point